app/src/main/java/com/infomaniak/drive/views/ShareLinkContainerView.kt

```diff
@@ -45,44 +45,79 @@ class ShareLinkContainerView @JvmOverloads constructor(
     fun setup(
         file: File,
         shareLink: ShareLink? = null,
-        onTitleClicked: (shareLink: ShareLink?, currentFileId: Int) -> Unit,
-        onSettingsClicked: (shareLink: ShareLink, currentFile: File) -> Unit
+        onTitleClicked: ((shareLink: ShareLink?, currentFileId: Int) -> Unit)? = null,
+        onSettingsClicked: ((shareLink: ShareLink, currentFile: File) -> Unit)? = null,
     ) {
         currentFile = file
         this.shareLink = shareLink
         isVisible = true
         urlValue = file.shareLink ?: ""
 
-        updateUi()
+        selectUI(file.isDropBox())
```

Comment:
I think that a comment, it'll be better, just say for example `Hide this component for dropbox folder` 🤔, or a method that is more meaningful